src/Safe7579Launchpad.sol

```diff
@@ -182,7 +185,18 @@ contract Safe7579Launchpad is IAccount, SafeStorage {
         }
 
         // initialize validator on behalf of the safe account
-        ISafe7579(initData.safe7579).launchpadValidators(initData.validators);
+        // the call below is equivalent to:
+        // ISafe7579(initData.safe7579).launchpadValidators(initData.validators);
+        // but we need to append msg.sender (entrypoint) to ERC2771 style access control, to protect
+        // the launchpadValidator function
+        (bool success,) = address(initData.safe7579).call(
```

Comment:
adding ERC2771 style appending of msg.sender to calldata. this should offer sufficient frontrun protection for the launchpadValidators function